src/v2i-hub/CDASimAdapter/src/CDASimAdapter.cpp

```diff
@@ -18,12 +18,12 @@ namespace CDASimAdapter{
         success = GetConfigValue<double>("X", location.X);
         success = success && GetConfigValue<double>("Y", location.Y);
         success = success && GetConfigValue<double>("Z", location.Z);
-        PLOG(logINFO) << "Location of Simulated V2X-Hub updated to : {" << location.X << ", " 
+        PLOG(logINFO) << "Location of Simulated V2X-Hub updated to : {" << location.X << ", "
```

Comment:
Why do all you PRs have changes in random lines of code. There is no change here right?

Reply:
Do you have some kind of formatting running

Reply:
Yeah it is the vscode whitespace formatter. if you turn it off in the github to see then it moves it away